src/main/java/org/opensearch/neuralsearch/processor/InferenceProcessor.java

```diff
@@ -51,14 +52,14 @@ public abstract class InferenceProcessor extends AbstractProcessor {
     private final Environment environment;
 
     public InferenceProcessor(
-        String tag,
-        String description,
-        String type,
-        String listTypeNestedMapKey,
-        String modelId,
-        Map<String, Object> fieldMap,
-        MLCommonsClientAccessor clientAccessor,
-        Environment environment
+            String tag,
+            String description,
+            String type,
+            String listTypeNestedMapKey,
+            String modelId,
+            Map<String, Object> fieldMap,
+            MLCommonsClientAccessor clientAccessor,
+            Environment environment
     ) {
```

Comment:
remove such diffs where there is no change in the code. This will ensure that only lines which are changed are present in the diff.